include/variables.h

```diff
@@ -280,21 +280,22 @@ extern char D_800994B0[];
 extern f64 D_800994B8;
 extern f64 D_800994C0;
 extern f64 D_800994C8;
-extern char D_800994D0[];
-extern char D_800994DC[];
-extern StackEntry sBootThreadInfo;
-extern OSThread sIdleThread;
-extern u8 sIdleThreadStack[1024];
-extern StackEntry sIdleThreadInfo;
-extern u8 sBootThreadStack[1024];
-extern IrqMgr gIrqMgr;
-extern u8 sIrqMgrStack[1280];
-extern StackEntry sIrqMgrStackInfo;
-extern OSThread gMainThread;
-extern u8 sMainStack[2304];
-extern StackEntry sMainStackInfo;
-extern OSMesg sPiMgrCmdBuff[50];
-extern OSMesgQueue gPiMgrCmdQ;
+extern char gBuildTeam[];
+extern char gBuildDate[];
+extern char gBuildMakeOption[];
+// extern StackEntry sBootThreadInfo;
```

Comment:
Any reason to not just delete these instead of commenting them out?

Reply:
There's a lot of commented-out stuff in this file. My assumption is that it's better to keep it around "just in case".